### PR TITLE
#41 - Verify digest from index.yaml

### LINKED ISSUE #41: IndexYaml.java:149-151: Create a unit test for digest...
The puzzle `32-66701d6b` from #32 has to be resolved: 

https://github.com/artipie/helm-adapter/blob/1aa237505bc1952d4a2c8fc9c579db750cfb5dec/src/main/java/com/artipie/helm/IndexYaml.java#L149-L151

The puzzle was created by @Sammers21 on 15-Jun-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by genryxy**
Closes #41 
Add test to verify digest from `index.yaml`

### CODE REVIEW

File: src/test/java/com/artipie/helm/IndexYamlTest.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            ).asciiString().toCompletableFuture().join()
+        );
+        MatcherAssert.assertThat(
+            (String)
+                ((ArrayList<LinkedHashMap<String, Object>>)
+                    ((Map<String, Object>)
+                        index.get("entries"))
+                        .get("tomcat"))
+                    .get(0)
+                    .get("digest"),
```
Comment (olegmoz) [inline src/test/java/com/artipie/helm/IndexYamlTest.java:95]: @genryxy this casts are really hard to read, I'd suggest to extract intermediate  variables to make it more readable

**Review by olegmoz — CHANGES_REQUESTED** [review]
@genryxy thanks, please see one suggestion

**Comment by genryxy** [comment]
@olegmoz thanks, extracted variables

**Review by olegmoz — APPROVED** [review]
@genryxy thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/helm-adapter#76` is not assigned, can't get performer

**Pull request merged**